src/main/webapp/app/overview/course-exercises/course-exercises.component.ts

```diff
@@ -15,15 +16,18 @@ import { Exercise, ExerciseService } from 'app/entities/exercise';
 export class CourseExercisesComponent implements OnInit, OnDestroy {
     public readonly DUE_DATE_ASC = 1;
     public readonly DUE_DATE_DESC = -1;
+    public readonly OVERDUE = 0;
+    public readonly NEEDS_WORK = 1;
     private courseId: number;
     private paramSubscription: Subscription;
     private translateSubscription: Subscription;
+    private filters: Set<number>;
```

Comment:
Nitpick: I think it would be better called `activeFilters`.

Reply:
Renamed the variable